From 8320cdf27912f1012102ebd9830b737d01c00b91 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Sat, 5 Jun 2021 22:58:47 +0200
Subject: [PATCH 01/13] Add desktop-id to Appstream metadata

It's needed for tools like appstream-generator to detect the associated desktop file.
---
 gobby-0.5.metainfo.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gobby-0.5.metainfo.xml b/gobby-0.5.metainfo.xml
index d99f57e..afa8fd3 100644
--- a/gobby-0.5.metainfo.xml
+++ b/gobby-0.5.metainfo.xml
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
   <id>de.0x539.gobby</id>
+  <launchable type="desktop-id">gobby-0.5.desktop</launchable>
   <metadata_license>CC0-1.0</metadata_license>
   <provides>
     <binary>gobby-0.5</binary>
-- 
2.30.2

